facebook-commerce.php

```diff
@@ -2493,16 +2493,8 @@ public function get_product_description_mode() {
 	 * @return string
 	 */
 	public function get_messenger_locale() {
-		/**
```

Comment:
I assume that you are not deleting this because it is a public interface.

My gut feeling is that we could still delete this and others as well. The filters are the integration points for merchants. Having the methods exposed was a mistake. 

I don't mind playing it safe so if we want to keep those let's keep them.

Reply:
Thanks, yeah, that's the concern I was going to ask about. I would be surprised if anyone was actually using these methods, but for safety I would leave them for this release, and then remove them in a following release (to give at least a couple weeks to catch the warning).

Reply:
See 
- https://github.com/woocommerce/facebook-for-woocommerce/issues/2724